subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -2773,6 +2799,19 @@ static inline void event_enc_prep(struct ll_conn *conn)
 		/* resume data packet rx and tx */
 		conn->llcp_enc.pause_rx = 0U;
 		conn->llcp_enc.pause_tx = 0U;
+
+		/* If the encryption procedure interrupted another procedcure,
+		 * continue that procedure
+		 */
+		if (conn->llcp.encryption.pend_llcp_type != LLCP_NONE) {
```

Comment:
If encryption setup complete at PHY update instant, then PHY update is skipped due to `conn->llcp.encryption.pend_llcp_type` being unset.